framework/set/setETCDSnapshots.go

```diff
@@ -0,0 +1,38 @@
+package framework
+
+import (
+	"github.com/hashicorp/hcl/v2/hclwrite"
+	"github.com/rancher/tfp-automation/config"
+	"github.com/zclconf/go-cty/cty"
+)
+
+// setCreateRKE2K3SSnapshot is a function that will create a snapshot for a RKE2/K3S cluster.
```

Comment:
can you modify this comment to specify that it is being set in main.tf, not actually creating the resource in this function?